### PR TITLE
[SYCL] Add E2E invoke_simd 'smoke' test.

### PR DESCRIPTION
**Pull request by kbobrovs**
Signed-off-by: Konstantin S Bobrovsky <konstantin.s.bobrovsky@intel.com>

### CODE REVIEW

**Comment by kbobrovs** [comment]
@cperkinsintel, @llvm-reviewers-runtime - friendly ping

File: SYCL/InvokeSimd/invoke_simd_smoke.cpp [hunk 1]
```diff
@@ -0,0 +1,163 @@
+// The tests a basic E2E invoke_simd test checking that invoke_simd
+// compiles and executes correctly on GPU, where the SIMD target is a
+// ESIMD function.
+
+// TODO: enable on Windows once driver is ready
+// REQUIRES: gpu && linux
+// UNSPPORTED: cuda || hip
```
Comment (cperkinsintel) [inline SYCL/InvokeSimd/invoke_simd_smoke.cpp]: spelling: UNSUPPORTED:

**Review by cperkinsintel — DISMISSED** [review]
one misspelled lit directive ( UNSUPPORTED ), but otherwise LGTM

File: SYCL/InvokeSimd/invoke_simd_smoke.cpp (hunk 1 above)
Comment (kbobrovs) [reply SYCL/InvokeSimd/invoke_simd_smoke.cpp]: Oh, thanks for the catch!

Comment (kbobrovs) [inline SYCL/InvokeSimd/invoke_simd_smoke.cpp]:
```suggestion
// UNSUPPORTED: cuda || hip
```

**Comment by kbobrovs** [comment]
@cperkinsintel - updated, thanks, please take a look

**Review by cperkinsintel — APPROVED** [review]
LGTM

**Pull request merged**